service/instance.py

```diff
@@ -743,11 +723,8 @@ def update_status(esh_driver, instance_id, provider_uuid, identity_uuid, user):
     if esh_driver.provider.location.lower() == 'mock':
         return None
     # Convert & Update based on new status change
-    core_instance = convert_esh_instance(esh_driver,
-                                         esh_instance,
-                                         provider_uuid,
-                                         identity_uuid,
-                                         user)
+    convert_esh_instance(esh_driver, esh_instance, provider_uuid, identity_uuid,
```

Comment:
The linter made this change because this used to be assigned to variable `core_instance`, but that variable is never used. That makes sense, but I am just commenting because it also changed the multi-line syntax of the function call. Which way is preferred? It seems weird to me to just put one of the parameters on its own line, but I also dislike having each on its own line. Is something like this bad form?
```
convert_esh_instance(esh_driver, esh_instance, provider_uuid,
                     identity_uuid, user)
```

This is a little crazy but I kinda like it:
```
args = (esh_driver, esh_instance, provider_uuid, identity_uuid, user)
convert_esh_instance(*args)
```

Reply:
From my perspective, the style doesn't matter as much as consistency. There are some styles that feel better than others, but then occasionally you'll read some code and you'll just wish that it would have chosen **any** style, because its illegible.

So the `*args` makes for good use of space, but I'd prefer to leave style to a program that is pretty darn good at ensuring legibility and also eliminates any time spent debating personal aesthetic. *args and **kwargs have a place but they make for easy mistakes, like switching the param order, or not having enough args. Or the person who changes all instances of `convert_esh_instance` to the new interface doesn't see the code that built up a *args list.

My next work will be to format the codebase.

Reply:
Thank you for clarifying, I completely agree. I recently learned about using `*args` with a tuple so I just like doing it 🤓